berkeley-mobile/Common/DetailView/OverviewCardView.swift

```diff
@@ -172,6 +186,26 @@ class OverviewCardView: CardView {
         }
     }
     
+    //MARK: - Label Selectors
+    @objc func phoneNumberTapped(gesture: DetailTapGestureRecognizer) {
+        if (gesture.view as? UILabel) != nil {
+            // Kinda hacky - necessary since cannot present alert when Popover is visible (ex. on Resources page)
+            self.window!.rootViewController?.dismiss(animated: true, completion: nil)
+            
+            UIApplication.shared.windows.first!.rootViewController!.presentAlertLinkTelephone(title: "Are you sure you want to open Phone?", message: "Berkeley Mobile wants to call \(gesture.phoneNumber)", options: "Cancel", "Yes", phoneNumber: gesture.phoneNumber.trimmingCharacters(in: .whitespacesAndNewlines).replacingOccurrences(of: "-", with: ""))
+        }
+    }
+    
+    @objc func addressTapped(gesture: DetailTapGestureRecognizer) {
+        if (gesture.view as? UILabel) != nil {
+            // Kinda hacky - necessary since cannot present alert when Popover is visible (ex. on Resources page)
+            self.window!.rootViewController?.dismiss(animated: true, completion: nil)
+            
+            UIApplication.shared.windows.first!.rootViewController!.presentAlertLinkMaps(title: "Are you sure you want to open Maps?", message: "Berkeley Mobile wants to navigate to \(self.addressLabel.text!)", options: "Cancel", "Yes", lat: gesture.latitude, lon: gesture.longitude, name: self.addressLabel.text!)
```

Comment:
Some items have addresses but no longitude / latitude (e.g. Suicide Resource Prevention Center). Tapping the address for these items opens Maps to the middle of the ocean (0, 0). Maybe you could make all the properties (or just lat / lon) in `DetailTapGestureRecognizer` optional and do some quick nil checks at the top of these functions?